docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+
+## Using `helm-run`
+
+`helm-run` is a command line tool for running benchmarks.
+
+To run this benchmark using the HELM command-line tools, we need to specify **run spec descriptions** that describes the desired runs. For this example, the run spec descriptions are `mmlu:subject=anatomy,model=huggingface/gpt-2` (for anatomy) and `mmlu:subject=philosophy,model=huggingface/gpt-2` (for philosophy).
+
+Next, we need to create a **run spec configuration file** contining these run spec descriptions. A run spec configuration file is a text file containing `RunEntries` serialized to JSON, where each entry in `RunEntries` contains a run spec description. The `description` field of each entry should be a **run spec description**. Create a text file named `run_spec.conf` with the following contents:
+
+```
+entries: [
+  {description: "mmlu:subject=anatomy,model=huggingface_gpt-2", priority: 1},
+  {description: "mmlu:subject=philosophy,model=huggingface_gpt-2", priority: 1},
+]
+```
+
+We will now use `helm-run` to execute the runs that have been specified in this run spec configuration file. Run this command:
+
+```
+helm-run --run-specs run_spec.conf --local --suite v1 --max-eval-instances 10
+```
+
+The meaning of the additional arguments are as follows:
+
+- `--local` specifies not to use the Stanford CRFM API. You should always use `--local` unless you are a Stanford CRFM API user.
+- `--suite` specifies a subdirectory under the output directory in which all the output will be placed.
+- `--max-eval-instances` limits evaluation to only the first *N* inputs (i.e. instances) from the benchmark.
+
+`helm-run` creates an environment directory environment and an output directory by default.
+
+-  The environment directory is `prod_env/` by default and can be set using `--local-path`.
+-  The output directory is `benchmarking_output/` by default and can be set using `--output-path`.
+
+After running this command, navigate to the `benchmarking_output/runs/v1/` directory. This should contain a two sub-directories named `mmlu:subject=anatomy,model=huggingface_gpt-2` and `mmlu:subject=philosophy,model=huggingface_gpt-2`. Note that the names of these sub-directories is based on the run spec descriptions we used earlier, but with `/` replaced with `_`.
+
+Each output sub-directory will contain several JSON files that were generated during the corresponding run:
+
+- `per_instance_stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics for each instance (i.e. input).
```

Comment:
Move this down near stats.json for coherence

Reply:
Moved.